phases/phase-1/challenges/referred/kwunyeung

```diff
@@ -0,0 +1 @@
+liangping
```

Comment:
I'm sorry but the user `kwunyeung` did not referred you. Would you mind deleting this file?